src/views/ContributionAssistant.vue

```diff
@@ -98,6 +102,45 @@
           </v-row>
         </v-container>
       </v-tabs-window-item>
+      <v-tabs-window-item value="Summary">
+        <v-container>
+          <v-row>
+            <v-col>
+              <h3 class="mb-4">
+                Please wait for upload
+              </h3>
+              <v-progress-linear
+                v-if="addProofLoading"
+                color="info"
+                height="25"
+                stripped
+                indeterminate
+              >
+                <strong>Uploading proof...</strong>
+              </v-progress-linear>
+              <v-progress-linear
+                v-if="!addProofLoading"
+                v-model="numberOfPricesAdded"
+                :max="productPriceForms.length"
+                :color="productPriceForms.length == numberOfPricesAdded ? 'success' : 'info'"
+                height="25"
+                stripped
+              >
+                <strong>{{ numberOfPricesAdded }} / {{ productPriceForms.length }} prices added</strong>
+              </v-progress-linear>
+              <v-btn to="/dashboard" class="mt-4" :aria-label="$t('Common.Dashboard')" :disabled="productPriceForms.length != numberOfPricesAdded">
```

Comment:
these 3 disabled could be transformed into 1 computed value "done" for instance ? + the above progress. 
or simply replace with `addProofLoading` ?
when would numberOfPricesAdded be different from productPriceForms.length ? (sure if the API bugs at that very moment, but unlikely, and maybe overkill to manage errors right now ?)

Reply:
productPriceForms.length tracks the total number of prices to push to API
numberOfPricesAdded tracks the number of prices handled by API

Since prices are added one by one, the values are different during the upload process, which can take a few seconds for many prices.

addProofLoading only tracks the proof API, when it's done, there still are a lot of prices to be added.

Agreed that it should be refactored as a "done" computed value !